### ISSUE TITLE
ASUS RT-AC87U, folder write-protected

### ISSUE THREAD

**Issue by ZuloEU** [thuantran/dnscrypt-asuswrt-installer#53]
Hi

Issue to install dnscrypt-asuswrt because a folder is write-protected (line 12).

USERNAME@RT-AC87U:/tmp/home/root#_ wget --no-check-certificate  -O installer https://raw.githubusercontent
.com/thuantran/dnscrypt-asuswrt-installer/master/installer && sh installer ; rm installer
--2018-06-13 14:18:56--  https://raw.githubusercontent.com/thuantran/dnscrypt-asuswrt-installer/master/installer
Resolving raw.githubusercontent.com... 151.101.84.133
Connecting to raw.githubusercontent.com|151.101.84.133|:443... connected.
WARNING: cannot verify raw.githubusercontent.com's certificate, issued by '/C=US/O=DigiCert Inc/OU=www.digicert.com/CN=DigiCert SHA2 High Assurance Server CA':
  Unable to locally verify the issuer's authority.
HTTP request sent, awaiting response... 200 OK
Length: 28588 (28K) [text/plain]
Saving to: 'installer'

installer                100%[====================================>]  27.92K  --.-KB/s  in 0.01s

2018-06-13 14:18:57 (2.40 MB/s) - 'installer' saved [28588/28588]

installer: readonly: line 12: TARG_DIR: is read only
USERNAME@RT-AC87U:/tmp/home/root#

---> I try to install on a ASUS RT-AC87U (firmware Version 3.0.0.4.382.50010 (2018/01/25)). Use Cloudflare DNS (and want to use DNS over HTTPS (DoH)).

**Comment by TheUntouchable** [comment]
Are you using asuswrt-merlin firmware? I think it will not work with the stock one..

**Comment by ZuloEU** [comment]
Hi

I'm using asuswrt-merlin firmware 384.5 (stable/current) from 2018-05-13.

After a lot of work I finally had install "dnscrypt-asuswrt-installer" but still have some issue to get it work correct. I have therefore decided not to use it and will close this issues. Unfortunately. Maybe it's just me and others are more lucky.

**Comment by TheUntouchable** [comment]
For the dnscrypt installer, did you visit the corresponding forum for that?
https://www.snbforums.com/threads/release-dnscrypt-installer-for-asuswrt.36071/

The installer worked for me without any problems on my RT-AC88U

**Comment by ZuloEU** [comment]
I try to visit the corresponding forum without to find a solution for my problem. I could have created a question in there but I didn't.

After some work I got it install. But I could not get DNS to work over TLS/HTTPS/DOH. Maybe I did something wrong in the setup - I'm just losing my internet connection. Then I have to start over again.

Maybe I will make a question in the corresponding forum sometime.
